roles/directorscriptruner.role

```diff
@@ -0,0 +1,16 @@
+nuclear_role(
+  # FileWatcher, ConsoleLogHandler and Signal Catcher Must Go First and without it many roles do not run
+  extension::FileWatcher
+  support::logging::ConsoleLogHandler
+  # This must come first as it emits config which many roles depend on (e.g. SensorFilter, WalkEngine)
+  actuation::KinematicsConfiguration
+  platform::cm740::HardwareIO
```

Comment:
just a note that this will need to be updated ini the opencr pr with the cmake subcontroller variable